schemas/styles/csl-terms.rnc

```diff
@@ -7,11 +7,14 @@ div {
     terms.gender-assignable
     | terms.gender-variants
     | terms.locator
-    | 
+    |
+      ## Item types
+      item-types
```

Comment:
I don't think there's any benefit to adding the doc string there.

```suggestion
    | item-types
```

Reply:
Done